src/SeqCli/Cli/Commands/HelpCommand.cs

```diff
@@ -142,6 +147,23 @@ static void PrintMarkdownHelp(string executableName, CommandList orderedCommands
             else
                 Console.WriteLine($"### `{cmd.Metadata.Name}`");
             Console.WriteLine();
+            if (cmd.Metadata.Platforms != SupportedPlatforms.All ||
```

Comment:
This is one level of nesting more than is required.

Reply:
Both are pulled under the one quote `>` block :+1: